### PR TITLE
fix #99 in createInLocation and add delete server by id

### PR DESCRIPTION
**Pull request by BenjaminLeibinger**

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -0,0 +1,2719 @@
+{
```
Comment (LKaemmerling) [inline composer.lock:1]: Hey, please don't commit the composer.lock

**Review by LKaemmerling — CHANGES_REQUESTED** [review]
Looks good, thank you! However please don't commit the composer.lock.

File: composer.lock (hunk 1 above)
Comment (BenjaminLeibinger) [reply composer.lock:1]: Why shouldn`t we commit the composer.lock? This ensures, that every developer has the same dependency versions. Its also recommended from the composer docu. [Composer Doku](https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control)

**Comment by LKaemmerling** [comment]
Thank you!

**Pull request merged**